internal/guard/guard.go

```
@@ -0,0 +1,301 @@
+		written = append(written, out)
+		return nil
+	})
+	return written, err
+}
+
+func copyFile(src, dst string) error {
+	in, err := os.Open(src)
+	if err != nil {
+		return err
+	}
+	defer in.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `in.Close` is not checked

(errcheck)